Langs/0404.lang

```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=Chinese (Traditional) - Taiwan
+Translator=SiderealArt
+Version=2.2.5.0
+[Strings]
+Donate=贊助
+Check for Updates=檢查更新
+Your Windows 11 Compatibility Results are Below=以下是您的 Windows 11 相容性結果
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=透過 https://www.whynotwin11.org/ 前往 WhyNotWin11
+Results Based on Currently Known Requirements!=結果為目前已知的系統要求
+Translation by=譯者
+Architecture (CPU + OS)=架構 (CPU + OS)
+Boot Method=開機方式
+CPU Compatibility= CPU 相容性
+CPU Core Count=CPU 核心數
+CPU Frequency=CPU 基頻
```

Comment:
I think it is the best according to Microsoft's version.

![image](https://user-images.githubusercontent.com/30853081/123656604-a87b9c80-d862-11eb-93bc-81e060948122.png)

Reply:
OK, I'll change it.